### PR TITLE
Closes #55 - Scheduler can now load job classes from multiple locations.

### LINKED ISSUE #55: Scheduler cannot load all annotated job classes from multiple locations
Hi,
This issue is related or a duplicate of https://github.com/knowm/Sundial/issues/36
When the package to scan for annotated classes exists in multiple locations (e.g. two source roots: main and test going to separate output directories but both ending up on the classpath or a main and additional jar) then only one of those locations will be picked up and scanned.

The solution would seem to be to either modify existing method 
https://github.com/knowm/Sundial/blob/develop/src/main/java/org/quartz/classloading/ClassLoadHelper.java -> `URL getResource(String name);` or add a new one that would allow multiple resource locations `URL[] getResource(String name);` and use it for finding package locations to iterate over in: https://github.com/knowm/Sundial/blob/develop/src/main/java/org/knowm/sundial/plugins/AnnotationJobTriggerPlugin.java -> `public Set<Class<? extends Job>> getJobClasses(String pkgname)`

Please let me know if you would have some time to look into that in near future and if not then I can try to submit a PR.

### PR DESCRIPTION
**Pull request by ligasgr**

### CODE REVIEW

File: src/main/java/org/quartz/classloading/CascadingClassLoadHelper.java [hunk 1]
```diff
@@ -208,23 +244,25 @@ public InputStream getResourceAsStream(String name) {
       String relPath = pkgname.replace('.', '/').replace("%20", " ");
 
       // Get a File object for the package
-      URL resource = getResource(relPath);
-      if (resource == null) {
+      List<URL> resources = getResources(relPath);
+      if (resources.isEmpty()) {
```
Comment (timmolter) [inline src/main/java/org/quartz/classloading/CascadingClassLoadHelper.java]: resources.size()>0 is preferred as I ran into issues with isEmpty many years ago. Can't remember why anymore though.

Comment (timmolter) [reply src/main/java/org/quartz/classloading/CascadingClassLoadHelper.java]: or <1 in this case...

**Comment by ligasgr** [comment]
Hi There,

Would you mind releasing a version with this change included? It's been 10 months now since it's been merged and I would love to take in the official version to fix our issue.
Please let me know if I can help in any way...

Thanks,
Grzegorz

**Comment by timmolter** [comment]
Done, 2.2.1. Thanks for the reminder...

**Comment by ligasgr** [comment]
Thanks a lot!
Roughly do you know how long it takes before it appears in https://mvnrepository.com/artifact/org.knowm/sundial ?

**Comment by timmolter** [comment]
not sure, but you can add it to your pom and it should already work.

**Pull request merged**